configuration/concepts/adult_hiv_initial_form.csv

```diff
@@ -0,0 +1,97 @@
+uuid,Members,Answers,Fully specified name:fr,Fully specified name:en,Short name:en,Short name:fr,Data class,Data type,_version:1,_order:1000
+5f38b80a-d4e2-4623-bb15-6662c33ab4f9,,,Nombre d'enfant en charge,Number of children in charge,,,Misc,Numeric,,
+6221e521-79c9-4a28-b3e5-e96df371b855,,,Age du dernier enfant,Age of last child,,,Misc,Numeric,,
+b2031349-b60a-433d-880a-8668cc040d2f,,,Hétérosexuel,Heterosexual,,,Misc,N/A,,
+9801e8cd-da10-4c52-adee-27cd6939adbb,,,Bisexuel,Bisexual,,,Misc,N/A,,
+c540d003-cdb5-434c-baa4-0b2b3e5e3205,,,Homosexuel,Homosexual,,,Misc,N/A,,
+211f0857-61a3-4049-9777-374c4a592453,,b2031349-b60a-433d-880a-8668cc040d2f;9801e8cd-da10-4c52-adee-27cd6939adbb;c540d003-cdb5-434c-baa4-0b2b3e5e3205,Orientation sexuelle,Sexual orientation ,,,Misc,Coded,,
+b7648479-e480-4772-b646-a5abc07bc4f7,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Partenaire informé,Informed partner,,,Misc,Coded,,
+7acfafa4-f19b-485e-97a7-c9e002dbe37a,,,Positif,Positive,,,Misc,N/A,,
+718b4589-2a11-4355-b8dc-aa668a93e098,,,Négatif,Negative,,,Misc,N/A,,
+b09900a6-4646-4804-b460-63be72879a13,,,Ne sait pas,Do not know,,,Misc,N/A,,
+351d2ae2-597e-471d-859c-ac0f782e06ec,,Positive;Negative;Do Not Know,Statut Partenaire,Status Partner,,,Misc,Coded,,
+b9bcbad0-3d5d-43f8-9cb8-b0a80ea440c7,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Partage des résultats sérologiques,Sharing of serological results,,,Misc,Coded,,
+3b75cc72-924b-4d36-8891-95484287495e,,,Inconu,Unknown,,,Misc,N/A,,
+9d26a59c-fe56-4509-b931-e380303a72c3,,,Non renseigné,Not specified,,,Misc,N/A,,
+2d1bf419-16a9-44a7-a11c-98d39b170cec,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e;Unknown;9d26a59c-fe56-4509-b931-e380303a72c3,Couples discordants,Discordant couples ,,,Misc,Coded,,
+d2b400ea-7510-432c-9ea4-27ea88736bb7,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e; ,Utilisation systématique du préservatif,Condom use systematically,,,Misc,Coded,,

```

Comment:
@lauravignoli Could you please remove the trailing ";" at the end of `b497171e-0410-4d8d-bbd4-7e1a8f8b504e; ,`